### PR TITLE
56754

### PR DESCRIPTION
**Pull request by robinwpdeveloper**
Trac ticket: [https://core.trac.wordpress.org/ticket/56754](https://core.trac.wordpress.org/ticket/56754)

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/deprecated.php [hunk 1]
```diff
@@ -1150,13 +1150,13 @@ function comments_rss_link($link_text = 'Comments RSS') {
  * @see get_category_feed_link()
  *
  * @param bool $display
- * @param int $cat_ID
+ * @param int $cat_id
  * @return string
  */
-function get_category_rss_link($display = false, $cat_ID = 1) {
+function get_category_rss_link($display = false, $cat_id = 1) {
```
Comment (costdev) [inline src/wp-includes/deprecated.php]: Unfortunately, we aren't able to change the case in function parameters. Why?

PHP 8 introduced [named parameters](https://www.php.net/manual/en/functions.arguments.php#functions.named-arguments), which are case-sensitive.

This is what would happen on sites running PHP 8 containing plugins/themes that used named parameters: https://3v4l.org/di62v

**Review by costdev — CHANGES_REQUESTED** [review]
Thanks for the PR @robinwpdeveloper!

I've left a note inline that we can't change the case of a function parameter. This means that the changes in `src/wp-admin/includes/taxonomy.php` (Lines 189-196) can be made here. Can you update the PR to remove all other changes? :slightly_smiling_face:

While this PR is the same as one of the patches on the ticket, creating this PR helped to verify that no automated tests were affected by the change, so thanks for that! :+1:

**Comment by robinwpdeveloper** [comment]
Thanks @costdev 
Changes are made as per feedback.

**Review by costdev — APPROVED** [review]
Thanks for the updates @robinwpdeveloper! LGTM :+1:

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! Merged in [r55190](https://core.trac.wordpress.org/changeset/55190).

**Pull request closed without merging**